### ISSUE TITLE
Comparison with Tensorflow

### ISSUE THREAD

**Issue by jjihoon** [itdxer/neupy#207]
Hi,

As I am new to machine learning, I just wonder how Neupy is different from Tensorflow. To be more specific, how its performance differs from Tensorflow in terms of accuracy and calculation speed.

Also, would it be difficult to convert Neupy code to one in Tensorflow?

Thanks,

**Comment by itdxer** [comment]
In a few words, NeuPy is more like a highly level library for neural networks where you think more about connection between layers than about low-level operations. Tensorflow on the other had is much more flexible library. NeuPy uses Theano library in the backend (you can find some performance comparisons betweeen these libraries online). The problem is that in maybe 9-10 months Theano will stop supporting there library and I plan to change neupy's backend to Tensorflow. I'm already working on it, but it's a lot of work... Answering your question, there is no easy way convert it.
